README.md

```diff
@@ -129,7 +164,8 @@ SQL database. If you need an example, you can use `people` and `wishes` models.
 Your answer...
 
 ```
-Replace this with your answer
+Each schema defines what each columns it has in application's database.
```

Comment:
We wanted you to talk about how we can show a relationship between tables. For instance how a foreign key like `person_id` might be used to relate the wish table to the `people` table.